### ISSUE TITLE
Vocore does not work on new kernels. ENODEV on mmap

### ISSUE THREAD

**Issue by Spacefreak18** [Vonger/vocore2#56]
On newer kernels, i'm using 6.10 but it seems to go back farther, the vocore fbusb driver will not work even after getting the driver to compile.

When you attempt to mmap the device you will get ENODEV.

**Comment by Vonger** [comment]
https://github.com/Vonger/mpro_drm, tested works in 6.10

**Comment by Spacefreak18** [comment]
yeah, you can close this, because if i have to go through DRM and X/wayland, there's less of an advantage in my setup using the vocore anymore. This is good to know though.
